.github/workflows/lint.yml

```
@@ -23,3 +23,21 @@ jobs:
       - name: Run golangci-lint
         run: make lint
         if: env.GIT_DIFF
+
+  super-linter:
+    name: Run super-linter (Markdown)
```

Given that super-linter can actually run multiple linters (e.g. markdown, yaml, etc...), I propose we just call this `documentation-linter` and remove markdown from the name. In the future, we might also want to lint YAML for example too. No strong opinion on this though.

```suggestion
  documentation-linter:
    name: Run super-linter
```